rfcs/README.md

```diff
@@ -0,0 +1,75 @@
+
+# Roadmap to Open Source Release
+
+This document contains the outline of our Roadmap brainstorm session. The idea is to transform this outline into different RFCs and discuss them as separate Pull Requests.
+
+## Communication Channels
+
+### Web Presence
+
+- Appealing web site/Github pages.
+- Documentation
+    - Architecture
+        - **Use cases** helps defining the release workflow.
+        - **Release workflow diagram** helps outlining required tests.
+    - Operations manual
+        - How to install Shipper?
+        - How to manage Shipper?
+            - How do I include a cluster?
+            - How do I cordon a cluster?
+            - How do I drain a cordoned cluster?
+    - User manual
+        - Where do I look for problems
```

Comment:
Suggested edits:
- Where do I look for answers
- Where do I look for existing problems

Just for the sake of reducing the ambiguity :-)